src/System.Globalization.Calendars/tests/PersianCalendar/PersianCalendarTest.cs

```diff
@@ -2,49 +2,41 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
-using System.Globalization;
 using Xunit;
 
-namespace System.Globalization.CalendarsTests
+namespace System.Globalization.Tests
 {
-    public class PersianCalendarTest
+    public class PersianCalendarTests
     {
         [Fact]
-        public void TestCalendarConversion()
+        public void CalendarConversion()
         {
-            PersianCalendar cal = new PersianCalendar();
-            int i = 0;
-
-            while (i < s_dates.Length)
+            PersianCalendar calendar = new PersianCalendar();
+            for (int i = 0; i < s_dates.Length; i+= 6)
             {
                 DateTime date = new DateTime(s_dates[i + 3], s_dates[i + 4], s_dates[i + 5]);
 
-                Assert.True(cal.GetYear(date) == s_dates[i], String.Format("Calculated Persian Year {0} not equal the expected year {1}", cal.GetYear(date), s_dates[i]));
-                Assert.True(cal.GetMonth(date) == s_dates[i + 1], String.Format("Calculated Persian Month {0} not equal the expected Month {1}", cal.GetMonth(date), s_dates[i + 1]));
-                Assert.True(cal.GetDayOfMonth(date) == s_dates[i + 2], String.Format("Calculated Persian Day {0} not equal the expected Day {1}", cal.GetDayOfMonth(date), s_dates[i + 2]));
-
-                DateTime result = cal.ToDateTime(s_dates[i], s_dates[i + 1], s_dates[i + 2], 0, 0, 0, 0);
+                Assert.Equal(s_dates[i], calendar.GetYear(date));
```

Comment:
I think having the message string in the assert is very helpful when something fail. why we are  removing it?